dart/lib/src/sentry.dart

```diff
@@ -20,11 +27,19 @@ class Sentry {
   static Hub get currentHub => _hub;
 
   /// Initializes the SDK
-  static Future<void> init(OptionsConfiguration optionsConfiguration) async {
+  /// passing a [AppRunner] callback allows to run the app within its own error zone (`runZonedGuarded`)
+  /// https://api.dart.dev/stable/2.10.4/dart-async/runZonedGuarded.html
+  static Future<void> init(
+    OptionsConfiguration optionsConfiguration, [
+    AppRunner callback,
```

Comment:
```suggestion
    AppRunner appRunner,
```

Reply:
I like this idea and I think we should bring this into `SentryFlutter` too, as this is a nice add on but most probably opt-in, I already got questions about the `runZonedGuard` and some people prefer to have their own, including when running multiple crash reporting tools.
Can we make this optional on SentryFlutter as well?

Reply:
✅